src/rpm_ostree/mod.rs

```diff
@@ -89,9 +92,20 @@ impl Release {
             version: node.version,
             checksum: node.payload,
             age_index: Some(age),
+            is_oci: { scheme == OCI_SCHEME },
         };
         Ok(rel)
     }
+    pub fn get_pullspec_hash(&self) -> Result<String> {
+        let split: Vec<&str> = self.checksum.split(":registry:").collect();
```

Comment:
This works but requires the use of that specific rpm-ostree URL format, see above comment